### ISSUE TITLE
Crash on iOS 12.4.8 because Stripe SDK requires SwiftUI

### ISSUE THREAD

**Issue by tjarksaul** [stripe/stripe-ios#1828]
## Summary
When running an app with the Stripe SDK on iOS 12.4.8 (latest iOS for iPhone 6) the app crashes on launch because it requires SwiftUI:
```
dyld: Library not loaded: /System/Library/Frameworks/SwiftUI.framework/SwiftUI
  Referenced from: /private/var/containers/Bundle/Application/14E4D763-C793-491B-A7CA-95478E70F5AE/CarSharing.app/Frameworks/Stripe.framework/Stripe
  Reason: image not found
```

## Code to reproduce
See above

## iOS version
12.4.8

## Installation method
Cocoapods

## SDK version
21.4.0

## Other information
n/a

**Comment by davidme-stripe** [comment]
Hello @tjarksaul, thanks for filing this! Which version of Xcode are you running? There was a known issue in Xcode 11 which caused SwiftUI.framework to be linked incorrectly, but it should be fixed in Xcode 12. I'm trying to reproduce this in Xcode 12.2 and 12.5, and I'm unable to do so.

**Comment by tjarksaul** [comment]
Hello @david-stripe, I am using Xcode 12.5.

**Comment by vCrespoP** [comment]
+1 to this @david-stripe 

Also happening in Xcode 12.5.1

We do **not** have SwiftUI in our project, not even linked (iOS 12 does not support it though)

**Comment by SarannyaSalim** [comment]
+ 1 @davidme-stripe 

Facing same issue.

**Comment by davidme-stripe** [comment]
Sorry for the trouble — we still can't reproduce this locally, but we'll update the Podspec to forcibly weak-link SwiftUI. We plan to release this update within the next few days.

**Comment by vCrespoP** [comment]
Thank you @david-stripe

We'll try as soon as it's out and let you know the outcome.

**Comment by davidme-stripe** [comment]
The update was delayed by a few days. In the meantime, I've pushed a workaround to this branch:

```
pod 'Stripe', :git => 'https://github.com/stripe/stripe-ios.git', :branch => 'weak-swiftui'
```

Please try it out and let me know if it resolves the issue. Thanks!

**Comment by jaytrisw** [comment]
@da-stripe I installed from this branch and it seems to be good to go!  Thank you for your efforts here!

**Comment by vCrespoP** [comment]
Same here, thanks for the fast response & fix.

Cheers @david-stripe !

**Comment by vCrespoP** [comment]
When can we expect this to be merged @david-stripe ? 
We always want to keep our Stripe dependency updated to the last version and not a specific branch :)

Thanks again!

**Comment by davidme-stripe** [comment]
We released this in [version 21.7.0](https://github.com/stripe/stripe-ios/releases/tag/21.7.0).
